src/pykeen/training/training_loop.py

```diff
@@ -389,7 +389,11 @@ def _train(  # noqa: C901
         # Take the biggest possible training batch_size, if batch_size not set
         batch_size_sufficient = False
         if batch_size is None:
-            if self.automatic_memory_optimization:
+            # On CPU memory optimization doesn't work, since OOM on RAM will crash the entire host machine.
```

Comment:
This sounds more severe than it usually is :slightly_smiling_face: 

Maybe update it to

```
Using automatic memory optimization on CPU may result in undocumented crashes due to OS' OOM killer.
```